<launch>
	<node pkg="gazebo_ros" name="gazebo_launch" type="gzserver" args="/home/jon/catkin_ws/src/finderbot/FinderBot_Gazebo/simple_maze.world"/>

	<node pkg="finderbot" name="Slam" type="SLAM_node" output="screen" />

	<node pkg="finderbot" name="Dummy_Map" type="dummy_map_node" output="screen" />

	<node pkg="finderbot" name="Test_Path_Exec" type="test_path_exec_node" output="screen" />
</launch>